### PR TITLE
fix: fix prompt clicking flakiness, fix multiple snap key permissions

### PR DESCRIPTION
**Pull request by mpetrunic**
- added ability to accept multiple key request permissions
- fixed flakiness and timeouts around prompt clicking -> tests are faster now

**Short description of work done**

### PR Checklist

- [ ] I have run linter locally
- [ ] I have run unit and integration tests locally
  - [ ] Update configuration the newest version (readme and const)
- [ ] Rebased to master branch / merged master

### Changes

- _EXAMPLE - Removed basic auth from the controller_

### Issues

Closes #

### CODE REVIEW

File: test/flask/keys-snap/snap.manifest.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     }
   },
   "initialPermissions": {
-    "snap_getBip44Entropy_1": {}
+    "snap_getBip44Entropy_1": {},
+    "snap_getBip44Entropy_47": {}
```
Comment (BeroBurny) [inline test/flask/keys-snap/snap.manifest.json:17]: these methods are deprecated (`snap_getBip44Entropy_*`), please use [snap_getBip44Entropy](https://docs.metamask.io/guide/snaps-rpc-api.html#snap-getbip44entropy)

**Review by BeroBurny — CHANGES_REQUESTED** [review]

File: src/page.ts [hunk 2]
```diff
@@ -35,7 +35,17 @@ export interface DappeteerPage<P = unknown> {
...
+      detached: boolean;
+      hidden: boolean;
+      timeout: number;
+    }>
+  ): Promise<DappeteerElementHandle>;
+
+  waitForSelectorIsGone(
+    selector: string,
+    opts?: Partial<{ timeout: number }>
   ): Promise<DappeteerElementHandle>;
```
Comment (BeroBurny) [inline src/page.ts:49]: Why do we return an element if waiting for the element is gone?
Expected to get a `boolean`, for like if is gone or still available

Comment (mpetrunic) [reply src/page.ts:49]: yup, should be removed

**Review by BeroBurny — COMMENTED** [review]
LGTM but build is falling :cry:

**Review by BeroBurny — APPROVED** [review]
LGTM

**Pull request merged**